src/ThreeEditor/util/Materials/materials.ts

```diff
@@ -1,19 +1,28 @@
 export const COMMON_MATERIAL_ICRUS = [104, 276, 223, 179, 222, 6, 245, 170, 1000];
```

Comment:
Add chosen materials from the new ones to this array.

Reply:
done in [65d8e1e](https://github.com/yaptide/ui/pull/488/commits/65d8e1eccab1f68cab7a0794f8cb660999179a5e)